src/estree.ts

```diff
@@ -356,21 +356,23 @@ export interface CatchClause extends _Node {
 
 export interface ClassBody extends _Node {
   type: 'ClassBody';
-  body: (ClassElement | FieldDefinition)[];
+  body: (ClassElement | PropertyDefinition)[];
```

Comment:
This is okay, but I was wondering why the specs are using `PropertyDefinition`. This is part of object literal if you look at the ecma specs. And if you look at the specs for the fields `FieldDefinition` is mentioned there.
The specs mxing two things that isn't related

Reply:
What do you mean? There is no `FieldDefinition` in current estree spec.

Reply:
That is what I find confusing. It's in the field def .
`PropertyDefinition` has nothing to do with fields. Fields is a Class feature, right?

See the current ECMA specs. It's actually part of ObjectLiteral

https://tc39.es/ecma262/#prod-ObjectLiteral

Reply:
That's confusing.

Reply:
Maybe you can drop a comment regarding this in the ESTree repo? My head is elsewhere atm. Working on TS parsing. Maybe something to consider for Meriyah to support?